files-csharp/FastestWayToReadATextFileInCsharp/Tests/WaysToReadATextFileInCsharpTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using FastestWayToReadATextFileInCsharp;
+
+namespace Tests
+{
+    public class WaysToReadATextFileInCsharpTests
```

Comment:
A note with all of the tests. The test case method name states that the expectation is the return of a string, but the assertions are that the methods return expected text. I do believe the expected text assertion is the correct assertion, but the test names should be modified to reflect what we are asserting.